src/Stripe.net/Services/Terminal/Readers/ReaderCreateOptions.cs

```diff
@@ -9,5 +9,8 @@ public class ReaderCreateOptions : ReaderSharedOptions
 
         [JsonProperty("registration_code")]
         public string RegistrationCode { get; set; }
+
+        [JsonProperty("operator_account")]
+        public string OperatorAccount { get; set; }
```

Comment:
Same here, `operator_account` can be used in Reader update requests and should be declared in `ReaderSharedOptions` rather than `ReaderCreateOptions`.